app/controlplane/cmd/main.go

```diff
@@ -124,8 +127,24 @@ func main() {
 	}
 }
 
+// Load the available third party integrations
+// In the future this code will iterate over a dynamic directory of plugins
+// and try to load them one by one
+func loadIntegrations(l log.Logger) (integrations.Initialized, error) {
```

Comment:
This method will eventually load dynamically from for example a directory if/when we move to a plugin based system